tests/unit/test_opds_publication.py

```diff
@@ -193,7 +182,8 @@ def test_parseEditionToPublication_ReaderFlagFalse(self, testPubEls, mocker):
             languages=[None, {'iso_3': 'tst'}, {'iso_3': 'oth'}],
             date_created='testCreated',
             date_modified='testModified',
-            items=[mocker.MagicMock(links=[mocker.MagicMock(id='testID', url='testURL', media_type='testType', flags={'reader': False})])],
+            items=[mocker.MagicMock(links=[mocker.MagicMock(id='testID', url='testURL', media_type='testType', flags={'reader': False})],
+                                    rights=[mocker.MagicMock(source='testSource', license='testLicense', rightsStatement='testStatement')])],
```

Comment:
it would be good to add an `assert` to test that `addField` was called for this right below where we do `metadata.addField.assert_has_calls`